homework_4/betsapi/internal/domain/mappers/betMapper.go

```diff
@@ -0,0 +1,26 @@
+package mappers
+
+import (
+	dto "github.com/superbet-group/code-cadets-2021/homework_4/betsapi/internal/api/controllers/models"
+	storagemodels "github.com/superbet-group/code-cadets-2021/homework_4/betsapi/internal/infrastructure/sqlite/models"
```

Comment:
Either this:
```suggestion
	"github.com/superbet-group/code-cadets-2021/homework_4/betsapi/internal/api/controllers/models"
	storagemodels "github.com/superbet-group/code-cadets-2021/homework_4/betsapi/internal/infrastructure/sqlite/models"
```

Or this:
```suggestion
	controllermodels "github.com/superbet-group/code-cadets-2021/homework_4/betsapi/internal/api/controllers/models"
	storagemodels "github.com/superbet-group/code-cadets-2021/homework_4/betsapi/internal/infrastructure/sqlite/models"
```